frontend/src/App.js

```diff
@@ -33,6 +33,9 @@ import ImmunohistochemistryDashboard from "./components/immunohistochemistry/Imm
 import ImmunohistochemistryCaseView from "./components/immunohistochemistry/ImmunohistochemistryCaseView";
 import RoutedResultsViewer from "./components/patient/resultsViewer/results-viewer.tsx";
 import EOrderPage from "./components/eOrder/Index";
+import RoutineIndex from "./components/Reports/routine/Index.js";
+
```

Comment:
Could you try to remove unnecessary spaces.

Reply:
Hey, thanks @sherrif10 for the review . I appreciate your feedback and will surely look into in removing the unnecessary spaces!